.github/workflows/pr_test.yml

```diff
@@ -125,8 +125,8 @@ jobs:
           key: ${{ runner.os }}-test-go-mod-${{ hashFiles('**/go.sum') }}
       - name: "Install Rust"
         uses: dtolnay/rust-toolchain@1.83.0
```

Comment:
All of the `1.83.0` references in the Actions workflows can be changed back to `stable` as part of this PR.